atomic_physics/utils.py

```diff
@@ -66,30 +68,50 @@ def d2f_dB2(atom: ap.Atom, lower: int, upper: int, eps: float = 1e-4):
 
 
 def field_insensitive_point(
-    atom: ap.Atom, lower: int, upper: int, B_min: float = 1e-3, B_max: float = 1e-1
+    atom: ap.Atom,
+    lower: int,
+    upper: int,
+    B_min: Optional[float] = None,
+    B_max: Optional[float] = None,
+    *,
+    B0: float = 1e-8
 ):
     """Returns the magnetic field at which the frequency of a transition
     between two states in the same level becomes first-order field independent.
 
     :param atom: the atom to work with
     :param lower: index of the lower energy state
     :param upper: index of the higher-energy state
-    :param B_min: minimum magnetic field used in numerical minimization (T)
-    :param B_max: maximum magnetic field used in numerical maximization (T)
+
+    :param B_min: (UNUSED) minimum magnetic field used in numerical minimization (T)
```

Comment:
Given that these parameters aren't used any more, I think we can just remove them now rather than depreciating them.